mne_bids_pipeline/_config.py

```diff
@@ -101,7 +100,7 @@
 BIDS dataset.
 """
 
-exclude_runs: Optional[Dict[str, List[str]]] = None
+exclude_runs: Optional[dict[str, list[str]]] = None
```

Comment:
I think this is only supported in Python 3.10+

Reply:
Looks like 3.9+ I think which we have in our reqs?

https://docs.python.org/3.9/library/typing.html#type-aliases

Reply:
I think I found the PEP that says 3.9 https://peps.python.org/pep-0585/

Reply:
Oh, great! 😊 all good then!